### DISCUSSION TITLE
Is FreeRDP still a collaborative project ?

### DISCUSSION THREAD

**Discussion by hardening** [FreeRDP/FreeRDP#10628] [General]
Hi,

So I'm not very happy of how things goes in FreeRDP for some time now. I feel like some technical decisions are imposed without being discussed, and when you're unhappy about them, it happened that things were already merged and reverting seems out-of-question. 

This has been the case between me and @akallabeth for some months now on various subjects:

* what policy we have with asserts
* changes around the `rdpSettings` API
* check tools configuration
* warnings hunting

I feel some frustration with that, and I had the impression of imposed rules that nobody agreed on previously. I even have the impression that for example on asserts, we kinda agreed on something else previously (so no excessive asserts on non exposed static functions), but there's still PR comments that prone another policy.

But you know that was just a feeling and when you arguing with someone else I was putting that on an emotional sensation. But this morning, I've tried putting numbers on that feeling, especially the feeling of imposed, non discussed, decisions. So I tried various filter searches on github with last year's PR: 
* `is:pr is:merged author:akallabeth created:>2023-09-18`: total 506
* `is:pr is:merged author:akallabeth created:>2023-09-18 review:approved`: total 179

I've done the maths, and so the title of the discussion (my github filter skills can also be defective, but with a quick check lots of PRs have been merged without any approval).

**Comment by geiseri** [comment]
As someone that has more or less lurked here for the past 10 years I have to say there is a small kabal of developers that work on this project.  To be fair though there are probably 10 people on this planet that understand the RDP specifications, and I would argue most of them work on this project.  (I would say more, but I think some of the original Citrix developers have died of old age and the subtle discrepancies between mstscax, Mac, and Android versions prove few work at MS)  One thing that the KDE project pioneered was a Code of Conduct.  I can argue that FreeRDP is much bigger in reach than KDE, or even Mozilla so it might be best served by that.  I think the only rub is that there are so few people that actually are core this is very hard to have anything better than what you have now.  Please keep up the great work nonetheless!
